src/test/java/uk/gov/hmcts/reform/sscs/service/bulkscan/CcdCallbackHandlerTest.java

```diff
@@ -103,7 +103,7 @@ public void should_return_exception_record_and_errors_in_callback_response_when_
 
         when(caseTransformer.transformExceptionRecordToCase(caseDataCreator.exceptionCaseData()))
             .thenReturn(CaseTransformationResponse.builder()
-                .errors(ImmutableList.of("Cannot transform Appellant Date of Birth. Please enter valid date"))
+                .errors(ImmutableList.of("Cannot transformers Appellant Date of Birth. Please enter valid date"))
```

Comment:
did you mean to make this change? transformers are great and all, but the sentence doesn't make sense anymore 👅

Reply:
Ha, think this was intellij doing some global refactor. Will update